core/src/lib.rs

```diff
@@ -469,11 +482,23 @@ pub enum AttributeBitFlags {
 }
 
 pub mod attribute {
-    #[derive(Clone,Debug,Eq,Hash,Ord,PartialOrd,PartialEq)]
+    use TypedValue;
+
+    #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialOrd, PartialEq)]
     pub enum Unique {
         Value,
         Identity,
     }
+
+    impl Unique {
+        // This is easier than rejigging DB_UNIQUE_VALUE to not be EDN.
+        pub fn to_keyword(self) -> TypedValue {
```

Comment:
Why not `into_typed_value` or `From<Unique> for TypedValue`?

Reply:
Because there are _two_ `TypedValue`s that correspond to `Unique::Value`: `TypedValue::Ref(35)` and `TypedValue::typed_namespaced_keyword("db.unique", "value")`.